src/main/java/club/sk1er/patcher/config/PatcherConfig.java

```diff
@@ -918,6 +918,13 @@ public static int getInventoryScale() {
     )
     public static boolean transparentChat;
 
+    @Property(
+        type = PropertyType.SWITCH, name = "Transparent Chat When Closed",
+        description = "Remove the background from chat, only when the chat is closed.\n§eCan positively impact performance.",
```

Comment:
```suggestion
        type = PropertyType.SWITCH, name = "Chat Background When Open",
        description = "Add back the background when chat is open.",
```
Though this wording is also a bit awkward.